rustworkx/rustworkx.pyi

```diff
@@ -974,41 +976,41 @@ def graph_transitivity(graph: PyGraph, /) -> float: ...
 
 # Traversal
 
-_BFSVisitor = TypeVar("_BFSVisitor", bound=BFSVisitor)
-_DFSVisitor = TypeVar("_DFSVisitor", bound=DFSVisitor)
-_DijkstraVisitor = TypeVar("_DijkstraVisitor", bound=DijkstraVisitor)
+_BFSVisitor = TypeVar("_BFSVisitor", bound=BFSVisitor, default=BFSVisitor)
+_DFSVisitor = TypeVar("_DFSVisitor", bound=DFSVisitor, default=DFSVisitor)
+_DijkstraVisitor = TypeVar("_DijkstraVisitor", bound=DijkstraVisitor, default=DijkstraVisitor)
 
 def digraph_bfs_search(
     graph: PyDiGraph,
     source: Sequence[int] | None = ...,
-    visitor: _BFSVisitor | None = ...,
+    visitor: _BFSVisitor = ...,
 ) -> None: ...
 def graph_bfs_search(
     graph: PyGraph,
     source: Sequence[int] | None = ...,
-    visitor: _BFSVisitor | None = ...,
+    visitor: _BFSVisitor = ...,
```

Comment:
This is consistent with the behavior for the universal functions: https://github.com/Qiskit/rustworkx/blob/f182dc232cf972526e381cdb17536e411ad27616/rustworkx/__init__.pyi#L587

This was an artifact of #787 that we translated literally into the typing stubs. In practice this is not a breaking change because `None` errors at runtime anyway.